.github/issue_template.md

```diff
@@ -3,3 +3,4 @@
 ### Actual behavior
 
 ### Links and screenshots illustrating the problem
+* ''Please provide links to example locations in the map when creating screen shots. If you wish to avoid revealing your location, do not use an example nearby.''
```

Comment:
Can you clarify whether you are requesting that the additional text be dropped entirely?  Just want to make sure I'm meeting the ask.

Reply:
Yes, please drop the additional text. You could try to rewrite the heading to make it clearer that we need links and screenshots, but that's not required. Also, I can't think of better wording.

Reply:
Just updated the text to "Screenshots with links" which seems more clear that screenshots need to be accompanied with links rather than "screenshots" and "links" being two separate possible categories of report.  It makes it clear that an explicit location link is required which should hopefully prevent cases of trying to obscure locations.